src/server/oasisapi/portfolios/models.py

```diff
@@ -42,9 +45,41 @@ def get_absolute_reinsurance_info_file_url(self, request=None):
     def get_absolute_reinsurance_scope_file_url(self, request=None):
         return reverse('portfolio-reinsurance-scope-file', kwargs={'version': 'v1', 'pk': self.pk}, request=request)
 
-    def get_absolute_storage_url(self, request=None):                                                                                                                                                                                                                                                                           
+    def get_absolute_storage_url(self, request=None):
         return reverse('portfolio-storage-links', kwargs={'version': 'v1', 'pk': self.pk}, request=request)
 
+    def run_oed_validation(self):
+        # Model to settings.py ?
+        VALIDATION_CONFIG = [
```

Comment:
can we import or load `VALIDATION_CONFIG` instead of re-defining it here?

Reply:
I see it is defined in `serializers,py` already. It'd be good to have a single source of truth for this global variable.

Reply:
Good spot, Moved them both to Django settings as `PORTFOLIO_VALIDATION_CONFIG`